pyuvsim/tests/conftest.py

```diff
@@ -42,6 +42,8 @@ def setup_and_teardown_package():
 
 
 @pytest.fixture(autouse=True)
-def ignore_center_deprecation():
+def ignore_deprecation():
     warnings.filterwarnings('ignore', message='The default for the `center` keyword has changed.',
                             category=DeprecationWarning)
```

Comment:
This warning was removed in version 1.5 of pyuvdata (we're now on 2.0), so I think you can take this filter out.

Reply:
I've removed the filter. Glad this is fixed!